vouch_lib/tests/organization.rs

```diff
@@ -0,0 +1,154 @@
+use bson::doc;
+use serial_test::serial;
+use tokio;
+use vouch_lib::{
+    organization::{Organization, OrganizationForm, Bank},
+    user::{UserForm},
+};
+
+#[tokio::test]
+#[serial]
+async fn create_org() {
+    vouch_lib::test_setup().await;
+
+    let mut org = OrganizationForm {
+        id: None,
+        name: "Company XYZ".into(),
+        address: Some("AECS Layout".into()),
+        contact: Some("+919876543210".into()),
+    };
+
+    let org = org.save().await.unwrap();
+    assert_eq!(org.name, "Company XYZ".to_string());
+    assert_eq!(org.address, Some("AECS Layout".to_string()));
+    assert_eq!(org.contact, Some("+919876543210".to_string()))
+}
+
+#[tokio::test]
+#[serial]
+async fn edit_org() {
+    vouch_lib::test_setup().await;
+
+    let mut org = OrganizationForm {
+        id: None,
+        name: "Company XYZ".into(),
+        address: Some("AECS Layout".into()),
+        contact: Some("+919876543210".into()),
+    };
+
+    let mut org = org.save().await.unwrap();
+    org.name = "Company 1".into();
+    OrganizationForm::from(org).save().await.unwrap();
+
+    let org = Organization::filter(None, None).await.unwrap();
+
+    assert_eq!(org[0].name, "Company 1".to_string());
+}
+
+#[tokio::test]
+#[serial]
+async fn list_org() {
+    vouch_lib::test_setup().await;
+
+    let mut org = OrganizationForm {
+        id: None,
+        name: "Company XYZ".into(),
+        address: Some("AECS Layout".into()),
+        contact: Some("+919876543210".into()),
+    };
+
+    org.save().await.unwrap();
+
+    let response = Organization::filter(None, None).await.unwrap();
+
+    assert_eq!(response.len(), 1);
+    assert_eq!(response, vec![Organization::from_form(org).await.unwrap()]);
+}
+
+#[tokio::test]
+#[serial]
+async fn filter_org() {
+    vouch_lib::test_setup().await;
+    let mut org1 = OrganizationForm {
+        id: None,
+        name: "Company XYZ".into(),
+        address: Some("AECS Layout".into()),
+        contact: Some("+919876543210".into()),
+    };
+
+    let mut org2 = OrganizationForm {
+        id: None,
+        name: "Company 1".into(),
+        address: Some("HSR Layout".into()),
+        contact: Some("+919876543211".into()),
+    };
+
+    org1.save().await.unwrap();
+    org2.save().await.unwrap();
+
+    let response = Organization::filter(Some(doc! {"contact": "+919876543211"}), None)
+        .await
+        .unwrap();
+    assert_eq!(response.len(), 1);
+    assert_eq!(response, vec![Organization::from_form(org2).await.unwrap()]);
```

Comment:
If we do make above mentioned changes then we can do this here.
```suggestion
    assert_eq!(response, vec![org2]);
```